pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -97,6 +100,10 @@ public RepositoryData prepare() {
                 return milestone();
             case BUILD_CONFIGS:
                 return buildConfigs();
+            case OFFLINER_ONLY:
+                log.info("Generating only the offline manifest");
+                generateOfflinerManifest();
+                return null;
```

Comment:
I guess `IGNORE` was already returning `null`. It is not clear what `null` should mean. Maybe we shoiuld switch to using `Optional.ofNullable()`?

Reply:
+1 (or something better than just null generally), that RepoData object is then passed throughout and some places (e.g. generating shared content reports) do not check for a null on it

Reply:
@akoufoudakis I assume you've tested the OFFERLINER_ONLY` option and that the `null` here doesn't get propagated anywhere else? It's a bit hard to tell without any tests included in this PR.